taxi/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Driver)
+class DriverAdmin(admin.ModelAdmin):
```

Comment:
It's not correct.
more appropriately:
`class DriverAdmin(UserAdmin):`

Reply:
Thx)